libs/sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -478,6 +478,7 @@ createStmt
     : CREATE TABLE (IF NOT EXISTS)? table
         '(' tableElement (',' tableElement)* ')'
          partitionedByOrClusteredInto withProperties?                                #createTable
+    | CREATE TABLE table AS '(' query ')'                                            #createTableAs
```

Comment:
The "Compatibility" section of the respective PostgreSQL documentation about "CREATE TABLE AS" says [1]:

> `CREATE TABLE AS` conforms to the SQL standard. The following are nonstandard extensions:
> - The standard requires parentheses around the subquery clause; in PostgreSQL, these parentheses are optional.

According to @seut, this can be implemented like

```suggestion
    | CREATE TABLE table AS insertSource                                             #createTableAs
```

Can you adjust this bit here?

[1] https://www.postgresql.org/docs/current/sql-createtableas.html#id-1.9.3.86.9

Reply:
@amotl Thank you Andreas, will fix this